airflow/security/permissions.py

```diff
@@ -63,8 +63,8 @@
 DAG_PERMS = {ACTION_CAN_READ, ACTION_CAN_EDIT}
 
 
-def permission_name_for_dag(dag_id):
-    """Returns the permission name for a DAG id."""
+def resource_name_for_dag(dag_id):
```

Comment:
This is the _right_ name for this, pretty sure. Input @jhtimmins?

Reply:
Missed this. Yup!